mkt/site/middleware.py

```diff
@@ -111,7 +111,7 @@ def process_request(self, request):
             if stored_ov_lang.lower() in settings.LANGUAGE_URL_MAP:
                 ov_lang = stored_ov_lang
 
-        if 'lang' in request.REQUEST:
+        if 'lang' in request.GET:
```

Comment:
Maybe fireplace reviews use this, I don't remember if we care about the lang of the user reviews that are posted ?